schema/ethernet.yml

```diff
@@ -234,3 +242,129 @@ properties:
         - short
         - long
         default: long
+  lldp-interface-config:
+    type: object
+    description: Configurations of LLDP on a specified interface.
+    properties:
+      lldp-admin-status:
+        type: string
+        description: Enables LLDP transmit, receive, or transmit and receive mode on the specified port.
+        enum:
+          - receive
+          - transmit
+          - tx-rx
+          - disabled
```

Comment:
@mikehansen1970 @binnyjeshan
Just a comment for discussion: do you think maybe we have to skip / stop adding <disabled> stuff into schema and stick to
<if it's not present in config request, it should be treated as a default - disabled / or whatever default is>?

So like here if we'd push a config request with lldp-admin-status omitted, it should be disabled
Or it's a too <shallow> approach, if you may, or not _that_ obvious and transparent?

Reply:
@Cahb @binnyjeshan 
This is a workable approach as long as it is clear. It might be better to have it set to whatever the default is, I am not sure. For some things, disabled, or just not configured, makes sense.
At a higher level though, for example, if you send only 1 port in the ethernet block, would that implicitly disable all the other ports? That needs clarity.

Reply:
(small note)
Also, on the other note:
```
- receive
- transmit
- tx-rx
```
Should we maybe make them aligned:
```
- rx
- tx
- rx-tx
```

Reply:
Will incorporate this.